stdlib/websub/src/main/ballerina/src/websub/Module.md

```diff
@@ -489,9 +489,16 @@ public type WebhookListener object {
         return self.websubListener.__start();
     }
     
-    public function __stop() returns error? {
-        return self.websubListener.__stop();
+    public function __gracefulStop() returns error? {
+        return ();
     }
+
+    public function __immediateStop() returns error? {
+        http:Listener? sListener = self.serviceEndpoint;
+        if (sListener is http:Listener) {
+            return sListener.__immediateStop();
+        }
+        return ();
```

Comment:
This doesn't compile right? 

Please replace these with 
```ballerina
    public function __detach(service s) returns error?  {
        return self.websubListener.__detach(s);
    }
    
    public function __immediateStop() returns error? {
        return self.websubListener.__immediateStop();
    }

    public function __gracefulStop() returns error? {
        return self.websubListener.__gracefulStop();
    }
```

Reply:
Fixed